Cargo.toml

```diff
@@ -27,6 +27,7 @@ features = [
 
 [dependencies]
 calloop = "0.9.3"
+dirs = "2.0.2"
```

Comment:
Why 2.0.2? Isn't the latest version 4?

Reply:
lol, indeed. I think I somehow thought that it's the latest since docs.rs showing multiple of them and I have issues with crates.io from time to time recently.